.github/workflows/release-automation.yml

```diff
@@ -0,0 +1,41 @@
+name: Update flyteidl version
+on: 
+  workflow_dispatch:
+
+jobs:
+  update-flyte-releases:
+    name: Update Flyteidl version 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Update release
```

Comment:
Should we use something like this?
https://github.com/marketplace/actions/dependencies-autoupdate

Reply:
No, it's not required.....`peter-evans/create-pull-request` is a more mature GitHub action. And we are using it in flyte so let's use one common action for creating a pull request.